caniusepython3/__init__.py

```diff
@@ -28,7 +28,7 @@
     CPU_COUNT = 2
 
 
-def check(requirements_paths=[], metadata=[], projects=[]):
+def check(requirements_paths=None, metadata=None, projects=None):
```

Comment:
This change was by design, so please revert the changes to this file.

Reply:
Interesting. Typically when I see lists as defaults for keyword arguments, it has unexpected side effects. Why are we doing this here?

Reply:
The only possible side-effects are if you mutate those defaults, but that function treats those arguments as immutable, so there's no side-effects. It also means the whole function can treat those arguments as being iterables no matter what, while your change requires the extra lines to make sure there is some default (which could be wrong in the fact of e.g. an empty string).